queenbee/base/parser.py

```diff
@@ -5,20 +5,24 @@
 import re
 
 
-def _check_list(lst, folder):
+def _check_list(lst: list, folder: str):
+    # TODO: Ask Mostapha for docs
```

Comment:
Mostapha has no idea right now! He can add it later when starts using the library. 😁